gulpfile.js/config/webpack.js

```diff
@@ -38,6 +27,18 @@ module.exports = function(env) {
   }
 
   if(env !== 'test') {
+    // Tests don't need the app entry points
+    webpackConfig.entry = {
+      page1: [jsSrc + 'page1.js'],
+      page2: [jsSrc + 'page2.js']
+    }
+
+    webpackConfig.output = {
+      path: jsDest,
+      filename: env === 'production' ? '[name]-[hash].js' : '[name].js',
+      publicPath: publicPath
+    }
```

Comment:
Oh, I dunno that it'll specifically hurt to leave it. But whatever webpack defaults to is fine, so I figured I'd only add this custom stuff for the environments that need it.